app-api-common/recipe/src/main/java/org/codeforvictory/android/recipes/data/api/recipe/RecipeService.kt

```diff
@@ -0,0 +1,11 @@
+package org.codeforvictory.android.recipes.data.api.recipe
+
+import org.codeforvictory.android.recipes.data.api.recipe.model.ApiMealsResponse
+import retrofit2.http.GET
+import retrofit2.http.Query
+
+interface RecipeService {
+
+    @GET
+    suspend fun filter(@Query("a") area: String = "American"): ApiMealsResponse
```

Comment:
IMO naming this filter is odd.  I'd usually see it named after the return (getRecipies etc).  The filtering bit is an implementation detail of the passed query param

Reply:
Sure. I actually thought a bit about the naming but I didn't have a better suggestion at the time. I think `recipesBy(...)` would make sense since we always need to specify a filter no matter what (although I provided a default argument because otherwise the API doesn't work (it's a tradeoff of having to deal with it).

Reply:
Usually i see the patter `getX`.

Reply:
`get and set` names usually come from the Java world. As discussed offline with Daniel, we decided to go for `recipesBy(...)`. Do you like it better @Mr-Gibbs ?